extensions/eda/plugins/event_source/aws_sqs_queue.py

```diff
@@ -29,6 +6,51 @@
 import botocore.exceptions
 from aiobotocore.session import get_session
 
+DOCUMENTATION = r"""
+---
+short_description: Receive events via an AWS SQS queue.
+description:
```

Comment:
Here we should mention that it is supported all the authentication methods supported by boto library and point to
"https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html"